buildscripts/update-lobster.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# this script pulls the evergreen-ci/lobster repo and copies the files so that
+# they can be served by logkeeper
+# it should only be run from the base logkeeper directory
+set -o errexit
+
+SCRIPTS_DIR=buildscripts
+LOBSTER_REPO=https://github.com/evergreen-ci/lobster.git
+LOBSTER_DIR=.lobster-temp
+LOBSTER_ASSETS_DIR=build/lobster
+LOBSTER_STATIC_DIR=static
+LOBSTER_HTML=index.html
+LOGKEEPER_STATIC_DIR=public/static
+LOGKEEPER_JS_DIR=js
+LOGKEEPER_CSS_DIR=css
+LOGKEEPER_TEMPLATES_DIR=templates/lobster/build
+
+# clone lobster repo and change to it
+pushd $SCRIPTS_DIR
+rm -rf $LOBSTER_DIR
+git clone $LOBSTER_REPO $LOBSTER_DIR
+pushd .lobster-temp
```

Comment:
use `cd` here, so you only need to pop once

Reply:
also this is LOBSTER_DIR

Reply:
I'd actually leave the pushd, and rearrange the pushes so they're at the beginning of the blocks, and pops so their at the end.

Reply:
went with Sam's suggestion